catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -12,10 +12,14 @@
 
 import logging
 
+from requests.exceptions import HTTPError
+
 from common import constants
 from common.licenses import get_license_info
 from common.loader import provider_details as prov
-from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
+from providers.provider_api_scripts.provider_data_ingester import (
+    ProviderDataIngester,
+)
```

Comment:
I _think_ this was an accidental change because a comma was added. I tested this locally and it should pass linting:
```suggestion
from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
```

Reply:
Those trailing commas always cause problems -- annoying the linter doesn't catch that :/ Fixed!